friture/Levels.qml

```diff
@@ -75,6 +75,28 @@ Rectangle {
             Layout.alignment: Qt.AlignVCenter | Qt.AlignRight
             color: systemPalette.windowText
         }
+		
+		Text {
+            id: splValues
+            textFormat: Text.MarkdownText
```

Comment:
Could you please rebase your branch, and then move to StyledText here ? That's a change that has been merged to improve the display of the dB values.

Also, please set a monospace font here.